doc/manuals/user/geolocation.md

```diff
@@ -464,3 +464,65 @@ be Alcobendas.
 }
 EOF
 ```
+
+## Geo-located queries NGSIv2
+
+The [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/) defines a Geographical Query
+languague (based on `georel`, `geometry` and `coords` fields) that can be used also in NGSIv1 with the
+FIWARE::Location::NGSIv2 scope, e.g.:
+
+```
+(curl localhost:1026/v1/queryContext -s -S --header 'Content-Type: application/json' \
+    --header 'Accept: application/json' -d @- | python -mjson.tool) <<EOF
+{
+    "entities": [
+        {
+            "type": "City",
+            "isPattern": "true",
+            "id": ".*"
+        }
+    ],
+    "restriction": {
+      "scopes": [
+        {
+          "type" : "FIWARE::Location::NGSIv2",
+          "value" : {
+            "georel": [ "near", "minDistance:13500" ],
+            "geometry": "point",
+            "coords": [ [40.418889,-3.691944] ]
+          }
+        }
+      ]
+    }
+}
+EOF
+```
+
+```
+(curl localhost:1026/v1/queryContext -s -S --header 'Content-Type: application/json' \
+    --header 'Accept: application/json' -d @- | python -mjson.tool) <<EOF
+{
+    "entities": [
+        {
+            "type": "City",
+            "isPattern": "true",
+            "id": ".*"
+        }
+    ],
+    "restriction": {
+    "scopes": [
+        {
+          "type" : "FIWARE::Location::NGSIv2",
+          "value" : {
+            "georel": [ "equals" ],
+            "geometry": "polygon",
+            "coords": [ [0, 0], [24, 0], [0, 20], [0, 0] ]
+          }
+        }
+      ]
+    }
+}
+EOF
+```
+
+Please note that the 'coords' field of the complex value of the scope needs to be put **after** the field 'geometry'.
```

Comment:
For literal citing fields, we typically used code-style, i.e. using `coords` instead of 'coords' (the same for geometry).